src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -5028,56 +5047,56 @@ void Fishing_HandleOwnerDialog(Fishing* this, PlayState* play) {
             if (play) {}
 
             if (Message_GetState(&play->msgCtx) == TEXT_STATE_NONE) {
-                this->unk_15C = 0;
-                if (D_80B7A68C != 0) {
-                    D_80B7A688 = 1;
-                    D_80B7A68C = 0;
+                this->stateAndTimer = 0;
+                if (sIsOwnersHatHooked) {
+                    sOwnerHair = FS_OWNER_CAPPED;
+                    sIsOwnersHatHooked = false;
                 }
-                D_80B7E0AC = 0;
+                sFishingPlayingState = 0;
                 play->interfaceCtx.unk_260 = 0;
             }
             break;
 
         case 23:
-            D_80B7A674 = false;
+            sIsRodVisible = false;
             if (Actor_HasParent(&this->actor, play)) {
-                this->unk_15C = 24;
+                this->stateAndTimer = 24;
             } else {
                 Actor_OfferGetItem(&this->actor, play, GI_SCALE_GOLDEN, 2000.0f, 1000.0f);
             }
             break;
 
         case 24:
-            D_80B7A674 = false;
+            sIsRodVisible = false;
             if ((Message_GetState(&play->msgCtx) == TEXT_STATE_DONE) && Message_ShouldAdvance(play)) {
-                if (D_80B7E07C == 0) {
-                    this->unk_15C = 0;
+                if (sFishOnHandIsLoach == 0) {
+                    this->stateAndTimer = 0;
                 } else {
                     Message_StartTextbox(play, 0x409C, NULL);
-                    this->unk_15C = 20;
+                    this->stateAndTimer = 20;
                 }
             }
             break;
     }
 }
 
-static s16 D_80B7AFC4[] = { 0, 1, 2, 2, 1 };
+static s16 sOwnerBlinkIndex[] = { 0, 1, 2, 2, 1 };
 
 static Vec3f sStreamSfxPos = { 670.0f, 0.0f, -600.0f };
 
 static Vec3s sSinkingLureLocationPos[] = {
```

Comment:
LocationPos sounds redundant to me? 
```suggestion
static Vec3s sSinkingLureLocation[] = {